application/pr/viewer.jsx

```diff
@@ -159,10 +181,10 @@ export default class PRViewer extends React.Component {
                         <p>Some of the contributors in this pull request were not recognized as having made the required IPR commitment to make substantive changes to the specification in this repository.</p>
                         <p>To fix this situation, please see which of the following applies:</p>
                         <ul>
-                        <li>if the contribution  does not concern a normative part of a specification, or is editorial in nature (e.g. fixing typos or examples), the contribution can be marked as non-substantive with the button above - this requires to be logged-in in this system.</li>
+                        <li>if the contribution does not concern a normative part of a specification, or is editorial in nature (e.g. fixing typos or examples), the contribution can be marked as non-substantive with the button above - this requires to be logged-in in this system.</li>
                        <li>if the said contributor is a member of {groups}, they should <a href="https://www.w3.org/users/myprofile/connectedaccounts">link their W3C and github accounts together</a>.</li>
                         {wgDoc}
-                        <li>Otherwise, the group’s chairs will need to figure how to get the proper IPR commitment from the contributor</li>
+                        <li>Otherwise, the group’s team contacts will request the non-participant licensing commitments the contributor</li>
```

Comment:
this needs rephrasing (either "from the contributors" or "will request the contributors to sign the non-participant..."

Reply:
The "chairs" phrasing should probably be kept for CGs.

Reply:
Addressed in https://github.com/w3c/ash-nazg/pull/163/commits/c5291ccd6ce483b2e738e7c2d15962dae6776e19